### PR TITLE
feat: use unified translator to translate ApisixRoute v1

### PR DESCRIPTION
**Pull request by tokers**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

**Comment by tokers** [comment]
@gxthrj Please take a look when you have time.

File: pkg/ingress/apisix/tls_test.go [hunk 1]
```diff
@@ -69,46 +69,6 @@ spec:
 	assert.EqualValues(t, sslExpect.Group, ssl.Group, "group convert error")
 }
 
-func TestConvert_group_annotation(t *testing.T) {
```
Comment (gxthrj) [inline pkg/ingress/apisix/tls_test.go:72]: Why remove this test case

File: pkg/ingress/apisix/tls.go [hunk 2]
```diff
@@ -32,8 +31,6 @@ type ApisixTLSCRD configv1.ApisixTls
 func (as *ApisixTLSCRD) Convert(sc Secreter) (*apisix.Ssl, error) {
 	name := as.Name
 	namespace := as.Namespace
-	_, group := BuildAnnotation(as.Annotations)
-	conf.AddGroup(group)
```
Comment (gxthrj) [inline pkg/ingress/apisix/tls.go:36]: We also need to explain on the document that the use of ingress.class has changed and is consistent with the official k8s

Comment (tokers) [reply pkg/ingress/apisix/tls.go:36]: Will add to document in the `ApisixRoute v2alpha1 ` PR .

File: pkg/ingress/apisix/tls_test.go (hunk 1 above)
Comment (tokers) [reply pkg/ingress/apisix/tls_test.go:72]: The `ingress.class` now doesn't have the mean for multiple cluster.

**Review by gxthrj — APPROVED** [review]

**Pull request merged**